pkg/gcv/validator.go

```diff
@@ -116,14 +117,14 @@ func newCFClient(
 		opt(options)
 	}
 
-	driver := local.New(options.driverArgs...)
-	// Append driver option after creation
-	options.backendArgs = append(options.backendArgs, cfclient.Driver(driver))
-	backend, err := cfclient.NewBackend(options.backendArgs...)
+	driver, err := local.New(options.driverArgs...)
 	if err != nil {
-		return nil, fmt.Errorf("unable to set up Constraint Framework backend: %w", err)
+		return nil, fmt.Errorf("unable to create new driver: %w", err)
 	}
-	cfClient, err := backend.NewClient(options.clientArgs...)
+	// Append driver option after creation
+	options.backendArgs = append(options.backendArgs, cfclient.Driver(driver))
+	options.backendArgs = append(options.backendArgs, options.clientArgs...)
+	cfClient, err := cfclient.NewClient(options.backendArgs...)
```

Comment:
it looks like we're no longer needing to create a backend - would it be possible to eliminate `options.backendArgs`? I'm concerned that in the long run it'll get confusing to have references to a backend handing around.

Reply:
Removed